thirdeye-persistence/src/main/java/ai/startree/thirdeye/datalayer/bao/TaskManagerImpl.java

```diff
@@ -147,13 +148,15 @@ AND ref_id not in (select ref_id from task_entity where status = 'RUNNING')
    * This method has side effects on the task DTO, even if the acquisition attempt fails.
    * Re-fetch the taskDto if you need to ensure consistency with the persistence layer.
    */
+  @Deprecated
```

Comment:
why are we deprecating this method?

Reply:
it was already deprecated in the interface, it's only used by the legacy task acquisition logic
same for the other one

Reply:
`acquireNextTaskToRun`
should be used instead, it performs find and acquire in a single optimized way